README.md

```diff
@@ -0,0 +1,216 @@
+<a name="#readme-top"></a>
```

Comment:
- [ ] Your frontend app code has a lot of commented code across several files.  Kindly remove it in order to preserve the repo's cleanliness. 👍